backend/models/listing.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
+
+const ListingSchema = new mongoose.Schema({
+    listerID: {type: mongoose.Schema.Types.ObjectId, required: true},
+    name: {type: String, required: true, minlength: 5, maxlength: 30},
+    description: {type: String},
+    rental_price: {type: Number, required: true},
+    date: {type: Date},
+    pet_friendly: {type: Boolean},
+    active: {type: Boolean, default: true},
```

Comment:
Similarly, this can be required since we default to true and we change it to false via the backend (so it should raise red flags if we see null)